hpux/generate_wazuh_packages.sh

```diff
@@ -56,50 +52,42 @@ utils_and_dependencies() {
 }
 
 config() {
-  echo USER_LANGUAGE="en" > $CONFIG
-  echo USER_NO_STOP="y" >> $CONFIG
-  echo USER_INSTALL_TYPE="agent" >> $CONFIG
-  echo USER_DIR=$INSTALL >> $CONFIG
-  echo USER_DELETE_DIR="y" >> $CONFIG
-  echo USER_CLEANINSTALL="y" >> $CONFIG
-  echo USER_BINARYINSTALL="y" >> $CONFIG
-  echo USER_AGENT_SERVER_IP="MANAGER_IP" >> $CONFIG
-  echo USER_ENABLE_SYSCHECK="y" >> $CONFIG
-  echo USER_ENABLE_ROOTCHECK="y" >> $CONFIG
-  echo USER_ENABLE_OPENSCAP="y" >> $CONFIG
-  echo USER_ENABLE_ACTIVE_RESPONSE="y" >> $CONFIG
-  echo USER_CA_STORE="n" >> $CONFIG
+  echo USER_LANGUAGE="en" > ${configuration_file}
+  echo USER_NO_STOP="y" >> ${configuration_file}
+  echo USER_INSTALL_TYPE="agent" >> ${configuration_file}
+  echo USER_DIR=${install_path} >> ${configuration_file}
+  echo USER_DELETE_DIR="y" >> ${configuration_file}
+  echo USER_CLEANINSTALL="y" >> ${configuration_file}
+  echo USER_BINARYINSTALL="y" >> ${configuration_file}
+  echo USER_AGENT_SERVER_IP="MANAGER_IP" >> ${configuration_file}
+  echo USER_ENABLE_SYSCHECK="y" >> ${configuration_file}
+  echo USER_ENABLE_ROOTCHECK="y" >> ${configuration_file}
+  echo USER_ENABLE_OPENSCAP="y" >> ${configuration_file}
+  echo USER_ENABLE_ACTIVE_RESPONSE="y" >> ${configuration_file}
+  echo USER_CA_STORE="n" >> ${configuration_file}
 }
 
 compute_version_revision()
 {
-    wazuh_version=$(cat ${SOURCE}/src/VERSION | cut -d "-" -f1 | cut -c 2-)
-    revision="$(cat ${SOURCE}/src/REVISION)"
+  wazuh_version=$(cat ${SOURCE}/src/VERSION | cut -d "-" -f1 | cut -c 2-)
 
-    if [ "${READY_TO_RELEASE}" == "yes" ]; then
-        revision="${OPTIONAL_REVISION}"
-    else
-        revision="0.${revision}dev"
-    fi
-
-    echo $wazuh_version > /tmp/VERSION
-    echo $revision > /tmp/REVISION
+  echo ${wazuh_version} > /tmp/VERSION
+  echo ${wazuh_revision} > /tmp/REVISION
 
-    return 0
+  return 0
 }
 
 download_source() {
   echo " Downloading source"
-  #Download source
-  /usr/local/bin/curl -k -L -O https://github.com/wazuh/wazuh/archive/$BRANCH.zip
-  /usr/local/bin/unzip $BRANCH.zip
-  mv wazuh-* $SOURCE
+  /usr/local/bin/curl -k -L -O https://github.com/wazuh/wazuh/archive/${wazuh_branch}.zip
+  /usr/local/bin/unzip ${wazuh_branch}.zip
+  mv wazuh-* ${source_directory}
   compute_version_revision
 }
 
 check_version(){
-  VERSION=`cat $SOURCE/src/VERSION`
-  number_version=`echo "$VERSION" | cut -d v -f 2`
+  wazuh_version=`cat ${source_directory}/src/VERSION`
+  number_version=`echo "${wazuh_version}" | cut -d v -f 2`
```

Comment:
Only FYI, you could use `number_version=${wazuh_version:1}`

Reply:
Noted.